Modules/_ctypes/callproc.c

```diff
@@ -1072,6 +1072,11 @@ GetComError(HRESULT errcode, GUID *riid, IUnknown *pIunk)
 #define IS_PASS_BY_REF(x) (x > 8 || !POW2(x))
 #endif
 
+/*
+ * Max number of arguments _ctypes_callproc will accept.
```

Comment:
Please add "bpo-13097: " and explain that the limit is for alloca() call in _ctypes_callproc().